pkg/autoscaler/autoscaler/autoscaler_manager.go

```diff
@@ -216,3 +237,95 @@ func (am *autoScalerManager) updateTidbClusterAutoScaler(tac *v1alpha1.TidbClust
 	}
 	return err
 }
+
+func (am *autoScalerManager) syncMonitor(tc *v1alpha1.TidbCluster, tac *v1alpha1.TidbClusterAutoScaler) error {
+	if monitorRef := tc.Status.Monitor; monitorRef != nil {
+		autoTcList, err := am.getAutoScaledClusters(tac, []v1alpha1.MemberType{v1alpha1.TiDBMemberType, v1alpha1.TiKVMemberType})
+		if err != nil {
+			return err
+		}
+
+		monitor, err := am.tmLister.TidbMonitors(monitorRef.Namespace).Get(monitorRef.Name)
+		if err != nil {
+			return err
+		}
+
+		return am.updateTidbMonitorClusters(monitor.DeepCopy(), autoTcList)
+	}
+	return nil
+}
```

Comment:
I think the logics here is a bit complex. I think we can directly get the auto-scaling clusters of the target tidbcluster. Then update the tidbmonitor directly.